docs/getting_started.md

```diff
@@ -877,8 +877,6 @@ Generate an ECDSA key:
     mbedtls_psa_crypto_free();
 ```
 
-### More about the Mbed Crypto library
-
-More information on [Mbed Crypto](https://github.com/ARMmbed/mbed-crypto/).
+### More about the Mbed Crypto
```

Comment:
Replace "More about the Mbed Crypto" with "More about the PSA Crypto API" or "More about PSA Crypto".